src/test/java/plugins/LdapPluginTest.java

```diff
@@ -205,7 +249,7 @@ public void login_use_fallback_server() {
         int freePort = this.findAvailablePort();
         LdapDetails ldapDetails = new LdapDetails("", 0, ldapContainer.getManagerDn(), ldapContainer.getManagerPassword(), ldapContainer.getRootDn());
         // Fallback-Config: primary server is not running, alternative server is running docker fixture
-        ldapDetails.setHostWithPort("localhost:" + freePort + " localhost:" + ldapContainer.getPort());
+        ldapDetails.setHostWithPort(ldapContainer.getHost()+":" + freePort + " " + ldapContainer.getHost() + ":" + ldapContainer.getPort());
```

Comment:
@jtnord Unless I am misunderstanding something, I think this is not incorrect. You cannot guarantee either that there is no LDAP server running on localhost. Actually, if the docker container is being run on Linux, the LDAP server will be running on localhost. The only way to actually be sure that there is no Ldap server listening on a given server and a given port is to use a port where you know no Ldap server will be listening. In this case 65000. The Ldap container exposes 636 and 389.